### PR TITLE
language fixes, compb fixes, compc fixes

### PR DESCRIPTION
**Pull request by aseemavi**
language fixes, compb fixes, compc fixes

compc-queue-policy model is not required since ietf-queue-policy model covers exactly that. 
There are language fixes which were reviewed earlier.
compc model changes are done based on changes to the base model. Further compc related changes are still required.

### CODE REVIEW

File: draft/draft-ietf-rtgwg-qos-model.xml [hunk 1]
```diff
@@ -121,15 +121,15 @@
...
+      <t> This document defines a YANG <xref target="RFC6020"/>
       <xref target="RFC7950"/> model for Quality of Service (QoS)
       configuration and statistics parameters. Traffic Policy module 
       defines the basic building blocks to define a classifier, policy
       and target. QoS Action module defines QoS action related
       parameters. Traffic policy is augmented to include packet match
       and action parameters to define the Diffrentiated Services
       (DiffServ) policy, Queues policy and  Scheduler policy. Each
-      of these policies are defined as separate modules. The Diffserv
+      of these policies is defined as separate module. The Diffserv
```
Comment (mjethanandani) [inline draft/draft-ietf-rtgwg-qos-model.xml:132]: s/module/modules/

or undo the change, as you are still referring to multiple modules.

File: draft/draft-ietf-rtgwg-qos-model.xml [hunk 2]
```diff
@@ -222,45 +222,41 @@
...
+      <t>One or more packet conditioning functions may be applied
+      to a classifier which may drop, mark, or delay packets
+      These are defined in the ietf-qos-action module. A set of
+      classifiers with corresponding conditioning functions when 
+      arranged in order of priority represent a QoS policy.
+      These policies are defined in ietf-traffic-policy module.</t>
 
-      <t>A meter qualifies if the traffic arrival rate is based on
-      agreed upon rate and variability. A meter is modeled based on
+      <t>The Meter meters each packet and passes the packet and the
```
Comment (mjethanandani) [inline draft/draft-ietf-rtgwg-qos-model.xml:244]: The use of capital letter Meter and Marker is inconsistent with the rest of the document. We would have update classifiers to Classifiers etc. Also we are not referring to a particular kind of meter. I would revert back to using meter and marker.

Also, can we update the definition of meter to say what you were saying before, which is "The meter qualifies if the traffic arrival rate is based on an agreed upon rate, and variability, and passes the metering result to the marker."

File: draft/draft-ietf-rtgwg-qos-model.xml [hunk 3]
```diff
@@ -297,17 +293,18 @@
...
-      one DiffServ module.</t>
-
-      <t>ietf-traffic-policy module contains list of classifier entries
-      identified by a classifier entry name. Each entry MAY contain a list of
-      filter entries.  When no filter entry is present in a classifier
-      entry, it matches all traffic.</t>
-
-      <t>ietf-traffic-policy module also contains list of policy objects identified
-      by a policy name and policy type which MUST be provided. With different
+      <t>This document defines a base traffic policy module, a QoS
```
Comment (mjethanandani) [inline draft/draft-ietf-rtgwg-qos-model.xml:296]: I would suggest that we give names to the modules, and refer to them by their name. For example, you could call ietf-traffic-policy module as the Base Traffic Policy Module, and replace all references to ietf-traffic-policy module as Base Traffic Policy Module.

Comment (aseemavi) [reply draft/draft-ietf-rtgwg-qos-model.xml:296]: then, somewhere we have to define atleast these three modules:

ietff-traffic-policy:         Base Policy Module
ietf-diffserv-policy:       Diffserv Module
ietf-qos-action:             QoS Action Module

File: draft/draft-ietf-rtgwg-qos-model.xml (hunk 2 above)
Comment (aseemavi) [reply draft/draft-ietf-rtgwg-qos-model.xml:244]: I think there was a yang doctor comment to use the definition from existing RFC rather than defining our own..

File: draft/draft-ietf-rtgwg-qos-model.xml (hunk 3 above)
Comment (aseemavi) [reply draft/draft-ietf-rtgwg-qos-model.xml:296]: I have fixed in new PR for Issue #53

**Review by mjethanandani — APPROVED** [review]
